examples/flatland/feedforward/decentralised/run_apex.py

```diff
@@ -0,0 +1,123 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import functools
+from datetime import datetime
+from typing import Any, Dict
+
+import launchpad as lp
+import sonnet as snt
+from absl import app, flags
+
+from mava.components.tf.modules.exploration.exploration_scheduling import (
+    LinearExplorationScheduler,
+    apex_exploration_scheduler,
+)
+from mava.systems.tf import madqn
+from mava.utils import lp_utils
+from mava.utils.environments.flatland_utils import flatland_env_factory
+from mava.utils.loggers import logger_utils
+from mava.wrappers.environment_loop_wrappers import MonitorParallelEnvironmentLoop
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    "mava_id",
+    str(datetime.now()),
+    "Experiment identifier that can be used to continue experiments.",
+)
+flags.DEFINE_string("base_dir", "./logs", "Base dir to store experiments.")
+
+flatland_env_config: Dict = {
+    "n_agents": 3,
+    "x_dim": 30,
+    "y_dim": 30,
+    "n_cities": 2,
+    "max_rails_between_cities": 2,
+    "max_rails_in_city": 3,
+    "seed": 0,
+    "malfunction_rate": 1 / 200,
+    "malfunction_min_duration": 20,
+    "malfunction_max_duration": 50,
+    "observation_max_path_depth": 30,
+    "observation_tree_depth": 2,
+}
+
+
+def main(_: Any) -> None:
+    """Run program
+
+    Args:
+        _ (Any): Arguments
+    """
+
+    # Environment.
+    environment_factory = functools.partial(
```

Comment:
Any reason why you didn't use lp_utils.partial_kwargs instead ? (like below)
Also, `include_agent_info=False` is the default, so unless you want to make it very explicit that we set it to False, you don't have to put it here

Reply:
Good question. @KaleabTessera, is there any reason to use `functools.partial` instead of `lp_utils.partial_kwargs` as in robocup?